lib/simplecov-html/version.rb

```diff
@@ -1,7 +1,7 @@
 module SimpleCov
   module Formatter
     class HTMLFormatter
-      VERSION = "0.10.1"
+      VERSION = "0.10.1".dup
 
       def VERSION.to_a
```

Comment:
again, I think I'd rather think the definition of the method on the object itself then going the `dup` route :D